docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+version: "3.8"
+
+services:
+  server:
+    build: .
+    environment:
+      IMS_DB_HOST_NAME: mysql
+      IMS_DB_DATABASE: ims
+      IMS_DB_USER_NAME: ims
+      IMS_DB_PASSWORD: ims
+    ports:
+      - "8080:80"
```

Comment:
In the future I'd like to have `Caddy` running in front of the server & clients. This is handy  on some other projects I do because you can use HTTPS locally to avoid some `CORS` issues and get a little closer to a real-world-like network landscape.

Reply:
Sure, I don't have experience with `Caddy` (I would tend to use nginx) but it seems straight forward.

Reply:
Caddy is just very light weight and small for local dev setups. It's not production level like Nginx but it's fast and light for development. I can do that as a follow up.